doc/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docs/base:latest
+MAINTAINER Sven Dowideit <SvenDowideit@docker.com> (@SvenDowideit)
+
+# to get the git info for this repo
+COPY . /src
+
+# Reset the /docs dir so we can replace the theme meta with the new repo's git info
+RUN git reset --hard
```

Comment:
This is awfully dangerous to run in someone's repo that may have local changes. Yes, it runs in the container but, what if one wants to test local changes?